fastn-core/src/library2022/processor/lang_details.rs

```diff
@@ -0,0 +1,41 @@
+pub async fn process(
+    value: ftd::ast::VariableValue,
+    kind: ftd::interpreter::Kind,
+    doc: &ftd::interpreter::TDoc<'_>,
+    req_config: &mut fastn_core::RequestConfig,
+) -> ftd::interpreter::Result<ftd::interpreter::Value> {
+    let current_language =
+        req_config
+            .config
+            .package
+            .current_language_meta()
+            .unwrap_or(LanguageMeta {
+                id: "en".to_string(),
+                id3: "eng".to_string(),
+                human: "English".to_string(),
+                is_active: true,
+            });
+    let available_languages = req_config.config.package.available_languages_meta();
+    let result = LanguageData {
+        current_language,
+        available_languages,
+    };
+    doc.from_json(&result, &kind, &value)
+}
+
+#[derive(Default, Debug, serde::Serialize)]
+pub struct LanguageData {
+    #[serde(rename = "current-language")]
+    pub current_language: LanguageMeta,
+    #[serde(rename = "available-languages")]
+    pub available_languages: Vec<LanguageMeta>,
```

Comment:
Does this include `current-language`? I am okay with duplication. In general I would iterate through the languages.

Also note that we want to show the language in the order they were specified in main packages' `FASTN.ftd` file.

Reply:
Yes `available-languages` has all language including the current-language. I will fix the ordering.